src/repositories/mongo_config.py

```diff
@@ -1 +1,5 @@
-MONGO_CONNECTION_STRING = "mongodb+srv://db-vape-sensor-inventory:theMostSecurePassword?!@cluster-vape-sensor-inventory-fcfqt.mongodb.net/test?retryWrites=true&w=majority"
\ No newline at end of file
+import ssl
+
```

Comment:
update connection string to use `inventory` database instead of `test` as previous